src/Web/Features/Orders/GetMyOrders.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using System.Linq;
+using MediatR;
+using Microsoft.eShopWeb.ApplicationCore.Interfaces;
+using Microsoft.eShopWeb.ApplicationCore.Specifications;
+using Microsoft.eShopWeb.Web.ViewModels;
+using System.Collections.Generic;
+
+namespace Microsoft.eShopWeb.Web.Features.Orders
+{
+    public class GetMyOrders : IRequest<IEnumerable<OrderViewModel>>
```

Comment:
I'd probably keep to one class per file, unless you're using inner classes, which I'm not sure makes sense here. Your folder/namespace organization here should keep them together well enough, I think.